superset-frontend/src/views/CRUD/data/dataset/AddDataset/index.tsx

```diff
@@ -71,13 +71,17 @@ export default function AddDataset() {
   >(datasetReducer, null);
 
   const HeaderComponent = () => (
-    <Header setDataset={setDataset} datasetName={dataset?.dataset_name ?? ''} />
+    <Header
+      setDataset={setDataset}
+      title={dataset?.table_name ?? 'New dataset'}
+      schema={dataset?.schema ?? ''}
+    />
   );
 
   const LeftPanelComponent = () => (
     <LeftPanel
       setDataset={setDataset}
-      schema={dataset?.schema}
+      schema={dataset?.schema ?? ''}
```

Comment:
Since `schema` is optional as per the definition in `LeftPanelProps` wouldn't be sufficient to just send `dataset?.schema` ? without the empty string.

Reply:
Good call, that does make more sense! Fixed in [`this commit`](https://github.com/apache/superset/pull/21557/commits/7fa31ba609eadb2b9d01a884abc8da57a053ee79).